libs/src/sdk/api/playlists/PlaylistsApi.ts

```diff
@@ -245,6 +249,114 @@ export class PlaylistsApi extends GeneratedPlaylistsApi {
     }
   }
 
+  /**
+   * Publish a playlist
+   * Changes a playlist from private to public
+   */
+  async publishPlaylist(
+    requestParameters: PublishPlaylistRequest,
+    writeOptions?: WriteOptions
+  ) {
+    // Parse inputs
+    parseRequestParameters(
+      'publishPlaylist',
+      PublishPlaylistSchema
+    )(requestParameters)
+
+    // Fetch playlist
+    const playlistResponse = await this.getPlaylist({
+      playlistId: requestParameters.playlistId,
+      userId: requestParameters.userId
+    })
+    const playlist = playlistResponse.data?.[0]
+
+    if (!playlist) {
+      throw new Error(
+        `Could not fetch playlist: ${requestParameters.playlistId}`
+      )
+    }
+
+    return await this.updatePlaylist(
+      {
+        userId: requestParameters.userId,
+        playlistId: requestParameters.playlistId,
+        metadata: {
+          ...playlist,
+          isPrivate: false
```

Comment:
is that the assumption every time that it's not private?

Reply:
I think that's the goal of this method! :)